docassemble/assemblylinewizard/custom_values.py

```diff
@@ -12,3 +24,68 @@ class Object(object):
 # Could be a list, but a dict will match RESERVED_PLURALIZERS_MAP in the
 # places it's used, so maybe easier to treat them both the same.
 custom_values.people_plurals_map = {}
+
+# Filled in by load_org_specific. Is a dictionary of configs that each org using
+# the Weaver can set, per installation. The specific configs are:
+# * jurisdiction_dependency_choices: a collection of other yaml files 
+#   that the generated interview will include, specifically a jurisdiction
+#   It's a map from keys, what the user will see when selecting dependencies, that maps to
+#   a list of 3 items: the pypi dependency, the yaml include, and a boolean for default selection
+custom_values.org_specific_config = None
+
+def load_org_specific(all_custom_values=custom_values):
+  if all_custom_values.org_specific_config is None:
+    try:
+      all_custom_values.org_specific_config = objects_from_file('org_specific.yml')
```

Comment:
This question can be for later--but what is the way you envision someone loading/creating this org_specific.yml? Ideally they will not need to fork the repo.

We have the same issue for the output_patterns.yml file of course. We just may want to make it a parameter now to ease loading it when we're ready to add that UI.

Reply:
In the description: 

> Future work there will be a write an interview that writes that file with github organization info, extra dependencies to possibly add, defaults, and eventually, things from generator_constants.py like Court types.

I wasn't imaging it to really be primarily human editable. There'd be one per installation of the package, and you'd use that admin interview (not yet in) to edit / add to it. I guess we could have an area edit with just the whole text for the installation's config as a fallback / to transfer between servers. Forking is an option, but I don't think we should recommend that.

I am just realizing that I'm not sure if what would happen to the config when updating the weaver. As long as we don't include some default with the some name as the prod config file I don't think it should be an issue?